R/R-KernSmooth/Portfile

```diff
@@ -9,7 +9,7 @@ R.recommended       yes
 R.setup             cran cran KernSmooth 2.23-22
 revision            0
 maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
-license             Unlimited
+license             public-domain
```

Comment:
The information you quoted states that there is copyright on the KernSmooth code, so it's specifically not public domain. Probably likewise for the other ports where this change has been made. It may be that `Permissive` would be correct, but we really need some documentation or clarification from the R developers that states what Unlimited is supposed to mean in the context of R package licences.

Reply:
Well, let me change to `Permissive` then.

Reply:
@jmroot Done.

Could you help with squash and merge this then? If I squash from my end, it will remove info on passed CI, and it is pointless to run 4 hours of compilation for a non-functional change.

Reply:
Ok, that just won’t be done. Squashed and rebased, hopefully CI pass.